subsys/partition_manager/Kconfig

```diff
@@ -33,6 +33,13 @@ endif
 
 endmenu
 
+config PM_SUBSYS_ONLY
+	bool "Use the Partition Manager for single image builds"
+	help
+	  Use the Partition Manager feature to partition the device even if
+	  only a single image is included in the build. Note: This option must
+	  be set through the projects prj.conf.
```

Comment:
Why must it be set in `prj.conf` ?
Is it not suppose to work if a user is enabling NCS Bluetooth (and NUS Central) using menuconfig ?

Besides, defining a prompt with the setting, actually makes it appear in menuconfig.
So either the comment, or the Kconfig file is wrong.

Reply:
removed